### ISSUE TITLE
Bootstrap samples validation won't work

### ISSUE THREAD

**Issue by keesvandieren** [l0rdn1kk0n/wicket-bootstrap#1024]
Reproduction:

* clone project, checkout wicket-10.x-bootstrap-5.x
* Import in IDEA, run `de.agilecoders.wicket.Start` in dir `$MODULE_WORKING_DIR$`
* Go to `http://localhost:8080/validation/tooltip`
* Submit the form

Exception stacktrace shows up:

```
	... 61 common frames omitted
2023-12-13 12:30:54,461 [qtp2003891312-21] WARN [org.eclipse.jetty.server.HttpChannel] - /validation/tooltip
java.lang.NoClassDefFoundError: com/google/common/base/Strings
	at de.agilecoders.wicket.extensions.markup.html.bootstrap.form.validation.FeedbackMessageBehavior.onComponentTag(FeedbackMessageBehavior.java:43)
	at org.apache.wicket.Component.renderComponentTag(Component.java:3961)
	at org.apache.wicket.Component.internalRenderComponent(Component.java:2501)
```

**Comment by keesvandieren** [comment]
Guava is not on the classpath. It is a transitive dependency in bootstrap-extensions:

```
        <dependency>
            <groupId>com.google.javascript</groupId>
            <artifactId>closure-compiler-unshaded</artifactId>
            <optional>true</optional>
        </dependency>
```

Proposed solution: add guava as a direct dependency of bootstrap-extensions.

**Comment by martin-g** [comment]
Fixed with e2cb5709
